subsys/bluetooth/controller/ll_sw/nordic/lll/lll_df_types.h

```diff
@@ -78,10 +78,16 @@ struct lll_df_adv_cfg {
 #define IQ_SAMPLE_STATURATED_16_BIT ((int16_t)0x8000)
```

Comment:
```suggestion
#define IQ_SAMPLE_SATURATED_16_BIT ((int16_t)0x8000)
```
I suggest global replace of STATURATED with SATURATED

Reply:
Done